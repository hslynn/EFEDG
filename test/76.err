Running /share/soft/mvapich2-2.0.1_icc/bin/mpiexec.hydra -np 76 -ppn 26 -f /share/home/hslynn/.lsbatch/1546156868.208196.hostfile -env MV2_CPU_BINDING_POLICY scatter ./simplest
