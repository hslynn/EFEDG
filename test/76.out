Sender: LSF System <lsfadmin@c3u12n02>
Subject: Job 208196: <76> in cluster <hpc.lsec.cc.ac.cn> Exited

Job <76> was submitted from host <ln02> by user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
Job was executed on host(s) <36*c3u12n02>, in queue <batch>, as user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
                            <36*c2u15n03>
                            <4*c6u01n01>
</share/home/hslynn> was used as the home directory.
</share/home/hslynn/Einstein_DG/test> was used as the working directory.
Started at Sun Dec 30 16:02:23 2018
Results reported on Sun Dec 30 16:13:53 2018

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
mpijob -t mvapich2 ./simplest
------------------------------------------------------------

TERM_OWNER: job killed by owner.
Exited with signal termination: Killed.

Resource usage summary:

    CPU time :                                   78.27 sec.
    Max Memory :                                 391 MB
    Average Memory :                             371.57 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Processes :                              52
    Max Threads :                                52

The output (if any) follows:



PS:

Read file <76.err> for stderr output of this job.

